includes/Dashboard.php

```diff
@@ -128,5 +128,11 @@ public function enqueue_assets( $hook_suffix ) {
 			[],
 			$version
 		);
+
+		// Dequeue forms.css, see https://github.com/google/web-stories-wp/issues/349
+		wp_styles()->registered['wp-admin']->deps = array_diff(
```

Comment:
This code is repeated. Is there anyway we could move to a function or trait so it is reused.

Reply:
Valid point, although I don't see too much value in extracting these few lines to a separate function as it's not overly complex.